### PR TITLE
fix(feeds): Miscs fixes for feeds

### PR DESCRIPTION
**Pull request by dr1rrb**
Miscs fixes for feeds

### CODE REVIEW

File: src/Uno.Extensions.Reactive/Core/Axes/MessageAxis.T.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ internal MessageAxis(string name, Aggregator aggregate)
 	/// <param name="value">The metadata to encapsulate..</param>
 	/// <returns>The raw axis value.</returns>
 	[Pure]
-	public MessageAxisValue ToMessageValue(T? value)
+	public MessageAxisValue ToMessageValue(T value)
```
Comment (nickrandolph) [inline src/Uno.Extensions.Reactive/Core/Axes/MessageAxis.T.cs:38]: Why isn't T marked as nullable?

Comment (nickrandolph) [inline src/Uno.Extensions.Reactive/Core/Axes/MessageAxis.T.cs:38]: The naming for the From and To methods might be better as ToAxisValue and FromAxisValue

File: src/Uno.Extensions.Reactive.UI/Presentation/Bindings/Collections/NullObservableVector.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		IEnumerator IEnumerable.GetEnumerator() => new Enumerator();
+
+		/// <inhertidoc />
+		public void Add(T item) => throw new NotSupportedException();
+		
+		/// <inhertidoc />
+		public void Clear() => throw new NotSupportedException();
+
+		/// <inhertidoc />
+		public bool Contains(T item) => false;
```
Comment (nickrandolph) [inline src/Uno.Extensions.Reactive.UI/Presentation/Bindings/Collections/NullObservableVector.cs:45]: Is there a reason that Clear throws an exception whereas Contains returns false? Shouldn't Contains throw a NotSupportedException too? Same for CopyTo, Remove etc

**Comment by nickrandolph** [comment]
@dr1rrb do we need to get these fixed merged as part of the release this week?

File: src/Uno.Extensions.Reactive.UI/Presentation/Bindings/Collections/NullObservableVector.cs (hunk 2 above)
Comment (dr1rrb) [reply src/Uno.Extensions.Reactive.UI/Presentation/Bindings/Collections/NullObservableVector.cs:45]: This collection is flagged as `ReadOnly` so we throw exception in all methods that attempt to modify the collection (i.e. write), while we do allow "read" methods.

**Review by nickrandolph — APPROVED** [review]

**Comment by dr1rrb** [comment]
Yes it is, but only impacts grouped collection which are not publicly exposed yet ... needs to rebase and merge

**Pull request merged**